src/constants/triggers.ts

```diff
@@ -32,39 +71,41 @@ export const taskRegexSelectors: RegexSelector[] = [
   },
 ];
 
+const failureTypeSubscriberOptions: string[] = [
+  "Any", // default
+  "Test",
+  "System",
+  "Setup",
+];
+
 export const failureTypeSubscriberConfig = {
   text: "Failure Type",
   key: "failure-type",
   type: "select",
-  options: {
-    any: "Any",
-    test: "Test",
-    system: "System",
-    setup: "Setup",
-  },
-  default: "any",
+  options: failureTypeSubscriberOptions,
 };
 
+const requesterSubscriberOptions: string[] = [
```

Comment:
This list is essentially repeated in the `getBuildInitiator()` function; can we resolve these two build initiator lists into one shared variable?

Reply:
done